src/integrationTest/resources/tests/arm/properties/all_properties/annotation-record.properties

```diff
@@ -0,0 +1,20 @@
+bf_001=OBJECT_TYPE
+bf_002=CASE_NUMBERS
+bf_003=FILE_TYPE
+bf_004=HEARING_DATE
+bf_005=CHECKSUM
+bf_006=TRANSCRIPT_REQUEST
+bf_007=TRANSCRIPT_TYPE
+bf_008=TRANSCRIPT_URGENCY
+bf_009=COMMENTS
+bf_010=CREATED_DATE_TIME
+bf_011=CREATED_DATE_TIME
```

Comment:
Should this be `START_DATE_TIME`?
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1725104200&preview=/1725104200/1753715590/image001.png#ARMLLD-Metadata

Reply:
This is the all properties so live and nle use 10 and 11 for CREATED_DATE_TIME. Its just to make sure that it tests all attributes and attributes that don't exist in the object don't end up getting set.